modules/client/test/integration/client/methods.test.ts

```diff
@@ -1691,5 +1695,284 @@ describe("Client", () => {
       //   "Invalid ENS name"
       // );
     });
+
+    describe("isDaoUpdateProposalValid", () => {
+      it("should check if a proposal is valid for updating the dao and fail with an invalid proposal id", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new Client(ctx);
+        const proposalId = TEST_DAO_ADDRESS;
+        expect(
+          () => client.methods.isDaoUpdateProposalValid(proposalId),
+        ).rejects.toThrow(new InvalidProposalIdError());
+      });
+
+      it("should check if a proposal is valid for updating the dao and fail with a proposal that does not exist", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new Client(ctx);
+        const proposalId = TEST_MULTISIG_PROPOSAL_ID;
+        const mockedClient = mockedGraphqlRequest.getMockedInstance(
+          client.graphql.getClient(),
+        );
+        mockedClient.request.mockResolvedValueOnce({
+          iplugin: null,
+        });
+        expect(
+          () => client.methods.isDaoUpdateProposalValid(proposalId),
+        ).rejects.toThrow(new ProposalNotFoundError());
+      });
+      it("should check if a proposal is valid for updating the dao and fail with a proposal that has more than one action", async () => {
+        const ctx = new Context(contextParamsLocalChain);
+        const client = new Client(ctx);
+        const proposalId = TEST_MULTISIG_PROPOSAL_ID;
+        const initializeFromAction = client.encoding.initializeFromAction(
+          daoAddressV1,
+          {
+            previousVersion: [1, 0, 0],
+          },
+        );
+        const subgraphInitializeFromAction: SubgraphAction = {
+          to: initializeFromAction.to,
+          value: initializeFromAction.value.toString(),
+          data: bytesToHex(initializeFromAction.data),
+        };
+        const mockedClient = mockedGraphqlRequest.getMockedInstance(
+          client.graphql.getClient(),
+        );
+
+        mockedClient.request.mockResolvedValueOnce({
+          iproposal: {
+            id: TEST_MULTISIG_PROPOSAL_ID,
+            dao: {
+              id: daoAddressV1
+            },
+            actions: [subgraphInitializeFromAction, subgraphInitializeFromAction],
+          },
+        });
+
+        const validationResult = await client.methods
+          .isDaoUpdateProposalValid(
+            proposalId,
+          );
+        expect(validationResult.actions).toBe(false);
+        expect(validationResult.implementation).toBe(false);
+        expect(validationResult.initData).toBe(false);
+        expect(validationResult.version).toBe(false);
```

Comment:
Would it be possible to write the test so that only `expect(validationResult.actions).toBe(false);` and the rest is `true` as you do below?